### PR TITLE
Add insert cursor up/down commands

### PR DESCRIPTION
**Pull request by masad-frost**
Adds a very important shortcut that adds a cursor on the next/prev line.

[#]:fel

---
This diff is part of a [fel stack](https://github.com/zabot/fel)
<pre>
* <a href="1">#1 Add insert at each line</a>
* <a href="4">#4 Add insert cursor up/down commands</a>
* <a href="2">#2 Add codemirror/state to peer deps</a>
* <a href="3">#3 Fix copy-line-up/down existing on linux</a>
* master
</pre>

### CODE REVIEW

File: src/multiCursor.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const addCursorFactory =
+  (direction: 'up' | 'down'): Command =>
+  (view) => {
+    const forward = direction === 'down';
+    const current = view.state.selection.ranges;
+    const added = current.map((r) => view.moveVertically(r, forward));
+
+    view.dispatch({
+      selection: EditorSelection.create([...added, ...current], 0),
```
Comment (sergeichestakov) [inline src/multiCursor.ts]: FYI you can use `addRange` here: https://codemirror.net/6/docs/ref/#state.EditorSelection.addRange

prolly better to stick w CM built-ins for these sort of ops

File: src/multiCursor.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Command, EditorView } from '@codemirror/view';
+import { EditorSelection } from '@codemirror/state';
+
+const addCursorFactory =
```
Comment (sergeichestakov) [inline src/multiCursor.ts]: nooooo not a factory haha we're not writing Java here 😆 

maybe something like `createAddCursor()`

**Review by sergeichestakov — APPROVED** [review]
Awesome thanks! Couple minor comments 🖱️

File: src/multiCursor.ts (hunk 2 above)
Comment (masad-frost) [reply src/multiCursor.ts]: :shrug:

File: src/multiCursor.ts (hunk 1 above)
Comment (masad-frost) [reply src/multiCursor.ts]: There's many ways to write this, I don't think using `addRange` is more correct or built-in than this

Comment (masad-frost) [reply src/multiCursor.ts]: I love to please my reviewers, so there you go, `addRange` is in full force!

**Pull request merged**